skypy/halos/sampler.py

```diff
@@ -0,0 +1,75 @@
+"""Colossus halo mass sampler.
+
+This module facilitates the sampling of halos from
+`Colossus <http://www.benediktdiemer.com/code/colossus/>`_.
+
+Models
+======
+.. autosummary::
+    :nosignatures:
+    :toctree: ../api/
+
+    colossus_mass_sampler
+
+"""
+
+import numpy as np
+import colossus as colossus
+from colossus.lss import mass_function
+from scipy import integrate
+
+__all__ = [
+        'colossus_mass_sampler',
+           ]
+
+
```

Comment:
```suggestion
try:
   import colossus
except ImportError:
   HAS_COLOSSUS = False
else:
   HAS_COLOSSUS = True


```
Defines a variable `HAS_COLOSSUS` which is True/False depending on if colossus is installed

Reply:
Thanks for this suggestion!